pages/about.js

```diff
@@ -0,0 +1,34 @@
+import fs from "fs";
+import matter from "gray-matter";
+import PatternPage from "../components/templates/PatternPage";
+import dictionary from '../content/en/dictionary.json';
+
+export default function About(props) {
+
+    return (
+        <PatternPage t={dictionary} pattern={props.pattern}/>
```

Comment:
Using the pattern page doesn't quite make sense for the about pages (or similar for future)

Reply:
The pattern page simply renders the markdown and has the header and footer, which is the exact same as the about page. Could re-name the template however it doesn't seem to make sense for a one off page when the primary focus is displaying patterns.

Reply:
I don't want to block the merge because of this, but I do **STRONGLY** recommend we give the about page it's own template component, I just know it's going to cause issues (headaches) any time we edit the template. Happy to do it myself in another PR.